config/default.yml

```diff
@@ -148,6 +148,71 @@ Style/BarePercentLiterals:
     - percent_q
     - bare_percent
 
+Style/Blocks:
```

Comment:
We should probably rename this to `BlockDelimiters` or something like this. I guess I chose a pretty confusing name originally.